### PR TITLE
fixes output_folder in other disk [Windows]

### PR DESCRIPTION
**Pull request by ferchaure**
Hi guys, another Windows related issue. Using the cd command doesn't work if the folder is in other disk, a line like "E:" should be added to move to that dik as well. I haven't checked but probably other sorters have this issue

### CODE REVIEW

File: spikesorters/waveclus/waveclus.py [hunk 1]
```diff
@@ -220,9 +220,10 @@ def _run(self, recording, output_folder):
 
         if 'win' in sys.platform and sys.platform != 'darwin':
             shell_cmd = '''
+                {disk_move}
                 cd {tmpdir}
                 matlab -nosplash -wait -log -r run_waveclus
-            '''.format(tmpdir=tmpdir)
+            '''.format(disk_move=str(tmpdir)[:2], tmpdir=tmpdir)
```
Comment (alejoe91) [inline spikesorters/waveclus/waveclus.py:226]: @ferchaure why is this using the first two charachters of tmpdir? To get the disk? For exampl in this case let's say that `str(tmpdir)[:2]` is `D:`: Then does the command `D:` changes directory in Windows?

Comment (ferchaure) [reply spikesorters/waveclus/waveclus.py:226]: exactly. another annoying windows thing. I found a nice explanation [here].(https://superuser.com/questions/1565202/cannot-change-drive-letter-in-command-prompt-cmd-other-than-c)

Comment (alejoe91) [reply spikesorters/waveclus/waveclus.py:226]: Awesome! I didn't know :) 

Can I ask you a favor? Could you check for similar cases in the other sorters as well? So we fix it once and for all

Comment (ferchaure) [reply spikesorters/waveclus/waveclus.py:226]: ok, I'll take a look this week.

Comment (alejoe91) [reply spikesorters/waveclus/waveclus.py:226]: Thanks!!

Comment (ferchaure) [reply spikesorters/waveclus/waveclus.py:226]: done

**Comment by alejoe91** [comment]
@ferchaure forgot to merge this. Going now :)

**Pull request merged**